packages/x-charts-vendor/package.json

```
@@ -0,0 +1,62 @@
+{
+  "name": "@mui/x-charts-vendor",
+  "version": "7.8.0",
+  "description": "Vendored dependencies for MUI charts",
+  "author": "MUI Team",
+  "main": "./index.js",
+  "keywords": [
+    "data visualization",
+    "React",
+    "d3",
+    "charting"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mui/mui-x"
+  },
+  "license": "MIT AND ISC",
+  "exports": {
+    "./package.json": "./package.json",
+    "./d3-*": {
+      "types": "./d3-*.d.ts",
+      "import": "./es/d3-*.js",
+      "default": "./lib/d3-*.js"
+    }
```

I would assume these are the same, but more generic, since we are not only exporting the d3 packages. 

```suggestion
    "./*": {
      "types": "./*.d.ts",
      "import": "./es/*.js",
      "default": "./lib/*.js"
    }
```